Source/doom.cpp

```diff
@@ -35,18 +39,70 @@ int doom_get_frame_from_time()
 	return DoomQuestState / 1200;
 }
 
-void doom_alloc_cel()
+void doom_cleanup()
 {
-	pDoomCel = DiabloAllocPtr(229376);
+#ifdef HELLFIRE
+	void *v0; // eax
+	if (pDoomCel) {

```

Comment:
only the if is new the rest appear to be the MemFreeDbg macro

Reply:
`void doom_cleanup()
{
#ifdef HELLFIRE
	if (pDoomCel) {
		MemFreeDbg(pDoomCel);
	}
#else
	MemFreeDbg(pDoomCel);
#endif
}`
Tried this but isn't bin exact
![image](https://user-images.githubusercontent.com/14297035/64070738-f6281d80-cc68-11e9-8434-9d28e4dcaaf9.png)

Reply:
Seems adding `pDoomCel = 0;` after the macro makes it bin exact, changed it